tests/ContentTransformer/HtmlStylesRemoverTest.php

```diff
@@ -0,0 +1,276 @@
+<?php
+
+namespace EMS\CoreBundle\Tests\ContentTransformer;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use EMS\CoreBundle\ContentTransformer\HtmlStylesRemover;
+
+class HtmlStylesRemoverTest extends WebTestCase
+{
+    public function testRemoveSimpleInline()
+    {
+        $input = <<<HTML
```

Comment:
Isn't there a way to provide an array of inputs, and an array of outputs that is used by 1 testcase. (in console the number of tests will be the same, but you only have to program the flow once)

Reply:
However, not sure if it will be possible to "customize" the error message for each case, which is now handled through the function names.

Reply:
We should discuss about this, I have yet another change in my head.